### ISSUE TITLE
[OTHER] [Good First Issue Finder app] Fill out Playwright skeleton tests: highlighted item

### ISSUE THREAD

**Issue by bugITwhisperer** [EddieHubCommunity/good-first-issue-finder#378]
### What would you like to share?

Playwright skeleton test in the [Good First Issue Finder application](https://github.com/EddieHubCommunity/good-first-issue-finder) requires filling out - as per information [here](https://finder.eddiehub.io/docs/testing)

```
test.fixme('is the highlighted item in the left-menu is the correct one', async ({ page }) => {
  // navigate to the docs landing page /docs
  // check if the Introduction element in the left menu is holding the active class
});
```

The introduction.spec.ts [file](https://github.com/EddieHubCommunity/good-first-issue-finder/blob/main/tests/docs/introduction.spec.ts) has to be updated.

### Additional information
Once the issue is approved, please assign it to me.

**Comment by Cahllagerfeld** [comment]
Thanks for raising! Do you want to work on this? :)

**Comment by bugITwhisperer** [comment]
@Cahllagerfeld: it seems the PW skeleton tests are duplicates of the two first tests, so instead of fixing those 2 test.fixme, I had them removed in my PR.

Are there by any chance need more PW tests welcome to add?

**Comment by Cahllagerfeld** [comment]
Sure, you can create new ones, either with `fixme` or filled out ones 👍: Lets get that PR merged first though :)

**Comment by bugITwhisperer** [comment]
The issues have been solved, closing as done.
